targets/TARGET_Cypress/TARGET_PSOC6/pwmout_api.c

```diff
@@ -235,9 +235,14 @@ void pwmout_init(pwmout_t *obj, PinName pin)
 
 void pwmout_free(pwmout_t *obj)
 {
-    /* Does nothing because it is not called in the MBED PWMOUT driver
-    * destructor. The pwmout_init handles multiple calls of constructor.
-    */
+#if DEVICE_SLEEP && DEVICE_LPTICKER
+    if (!Cy_SysPm_UnregisterCallback(&obj->pm_callback_handler))
```

Comment:
Can you fix the style `{` on the same line (entire file is consistent)

Reply:
Done